opennms-provision/opennms-provisiond/src/main/java/org/opennms/netmgt/provision/service/ImportScheduler.java

```diff
@@ -36,12 +36,7 @@
 import org.opennms.core.utils.url.GenericURLFactory;
 import org.opennms.netmgt.config.provisiond.RequisitionDef;
 import org.opennms.netmgt.dao.api.ProvisiondConfigurationDao;
-import org.quartz.CronTrigger;
-import org.quartz.JobDetail;
-import org.quartz.JobKey;
-import org.quartz.Scheduler;
-import org.quartz.SchedulerException;
-import org.quartz.TriggerKey;
+import org.quartz.*;
```

Comment:
FYI, we try to always use full imports rather than wildcards, since it makes it easier to grep through the codebase for things using certain classes.

Reply:
Oops.. I think IntelliJ did that. Will fix.